DcCore/DcCore/Extensions/UIView+Extensions.swift

```diff
@@ -7,19 +7,19 @@ public extension UIView {
         self.layer.borderWidth = 2
     }
 
-    func alignLeadingToAnchor(_ anchor: NSLayoutXAxisAnchor, paddingLeading: CGFloat = 0.0, priority: UILayoutPriority? = .none) {
+    func alignLeadingToAnchor(_ anchor: NSLayoutXAxisAnchor, paddingLeading: CGFloat = 0.0, priority: UILayoutPriority = .required) {
```

Comment:
i am sure, it is correct, just asking for understanding: what is the change from `.none` to `.required` about? or was the default just never used?

Reply:
It's about the Optional in combination with the later `if let priority`:

The [default priority of a constraint](https://developer.apple.com/documentation/uikit/nslayoutconstraint/1526946-priority) is `required`. If you don't set/change it, it'll be `required`. So why using an Optional here if there's a default?